lib/BackendClient.js

```diff
@@ -437,6 +459,40 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
+    function createRecordedEntry(entryObj)  {
+        let deferred = Q.defer();
+        client.startMultiRequest();
+        // entryServerNode.listAction() parameter
+        let entryServerNodeFilter = new kalturaVO.KalturaEntryServerNodeFilter();
+        entryServerNodeFilter.entryIdEqual = entryObj.entryId;
```

Comment:
why are we filtering the entryServerNodeId?  don't we know that already?

Reply:
because after stop start the providing DC can change with RecordedEntryId. We need to filter EntryServerNodes having different RecordingEntryId's recordingInfo... we cannot filter in list request because BE doesn't support that